bin/osg-notify

```diff
@@ -144,6 +147,21 @@ def network_ok(bypass_dns_check):
         print("  https://opensciencegrid.org//operations/services/sending-announcements/")
         return False
 
+def replace_smart_quotes_and_dashes(contents):
+    # Replace smart quotes and em/en dashes
+    replaced = contents.replace('“','"').replace('”','"').replace("–","-").replace("—","-")
+    return replaced
+
+def has_non_ascii_characters(contents):
+    ret = False
+    # For each character in the message
+    for c in contents:
+        # Chek if the numeric value of the character is outside the ascii range (0-127)
+        if ord(c)< 0 or ord(c) > 127:
```

Comment:
If this was really the check you wanted, note that you can equivalently write it as
```
if not 0 <= ord(c) <= 127:
```

Though:
- `ord(c)` can't be `< 0`, so this part of the check is sort of meaningless
- the printable range is `32-126`; do you really want to allow `NUL` bytes (`0`), control characters (`1-31`) and ascii `DEL` (`127`) ?

Reply:
I've changed it to only allow the 32-126 range

Reply:
well that range and 10 which is '\n'

Reply:
and 9 which is a tab. Hopefully I'm not leaving any character that can be used out